tasks/codespell.rake

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+desc 'Run codespell over the files modified by the last commit'
+task :codespell do |_task|
+  next if Gem.win_platform?
+  next if ENV['CI'] # CI has its own workflow for this
+
+  sh 'which codespell &> /dev/null', verbose: false do |ok, _res|
+    next unless ok # codespell is optional
```

Comment:
Maybe warn that `codespell` is not installed at this point?

Reply:
Sure, that doesn't sound like a bad idea